streams/src/main/java/org/apache/kafka/streams/processor/internals/MinTimestampTracker.java

```diff
@@ -77,4 +77,8 @@ public long get() {
             return stamped.timestamp;
     }
 
+    public void clear() {
```

Comment:
Where do we call this?

Reply:
It is called in `RecordQueue.clear()`.

Reply:
Now it is :) You added some more commits :)

Reply:
That's right. I missed adding the diff when committing for the first time.